models/holidays.js

```diff
@@ -0,0 +1,56 @@
+var mongoose = require('mongoose');
+
+// Article Schema
+var holiSchema = mongoose.Schema({
```

Comment:
I think there should be another field for type of user.

Reply:
please elaborate

Reply:
like user type: where we add employee, admin, supervisor etc. So that if we have to use the user class we won't have to fetch the user. Depends on you, if you want to add it, it is not necessary

Reply:
it won't be holischema it will be Holiday